src/UI/MainMenuOptions.cpp

```diff
@@ -74,35 +74,45 @@ void MainMenuOptions::init()
 		const auto default_resolution = std::to_string(constants::MINIMUM_WINDOW_WIDTH) + 'x' + std::to_string(constants::MINIMUM_WINDOW_HEIGHT) + 'x' + std::to_string(60);
 		cmbResolution.text(default_resolution);
 	}
+
+    auto& r = NAS2D::Utility<NAS2D::Renderer>::get();
+    auto resolutions = r.getDisplayModes();
+    resolutions.erase(std::remove_if(std::begin(resolutions), std::end(resolutions),
+        [](const NAS2D::DisplayDesc& desc) {
+            const auto w = desc.width;
+            const auto h = desc.height;
+            const auto min_w = constants::MINIMUM_WINDOW_WIDTH;
+            const auto min_h = constants::MINIMUM_WINDOW_HEIGHT;
+            const auto is_invalid = w < min_w || h < min_h;
+            return is_invalid;
+        }),
+    std::end(resolutions));
```

Comment:
The formatting looks a bit inconsistent with the indentation for this block.

Reply:
I was trying to convey that the lambda went with and was part of the `eraase-remove` idiom calls.

What would you prefer?

Reply:
Well, to fully spill things, it might look like:
```cpp
	resolutions.erase(
		std::remove_if(
			std::begin(resolutions),
			std::end(resolutions),
			[](const NAS2D::DisplayDesc& desc) {
				const auto w = desc.width;
				const auto h = desc.height;
				const auto min_w = constants::MINIMUM_WINDOW_WIDTH;
				const auto min_h = constants::MINIMUM_WINDOW_HEIGHT;
				const auto is_invalid = w < min_w || h < min_h;
				return is_invalid;
			}
		),
		std::end(resolutions)
	);
```

----

On another matter, I don't think I would use all those `const` just to create a one time use short alias. It would be more straightforward to just use the original values in the comparison. No tracing of assignments to find where things came from. It makes the comparison line look a little longer, but even if that got spilled, it would still be shorter than creating the `const` aliases.

Reply:
Of course, if we merge that other PR for the comparisons, that would also really simplify the lambda.

Reply:
Oh god no. That's arrow code right there.

I have to split them up because I don't care about the refresh rate and I don't want any false positives from `DisplayDesc::operator<` removing descriptions that were okay to keep because the refresh rates were different.

Reply:
Technically, I did nothing to the actual structure of the code. The nesting is exactly the same as before. The cyclomatic complexity is exactly the same as before. If the newly formatted code is "arrow code", or deeply nested code, then so was the original. The indentation merely makes it more obvious.

----

More to the point, the original code had indentation that was contrary to the structure. The last line should not have been indented the same as the first line. The second last line mixes two levels of nesting in an inconsistent way.

Reply:
Fixed. It's only one extra line anyway.